package.json

```diff
@@ -63,19 +64,17 @@
     "simple-git": "^1.132.0"
   },
   "devDependencies": {
-    "@types/fs-extra": "^5.0.4",
-    "@types/fs-extra-promise": "^1.0.7",
+    "@types/fs-extra": "^8.1.0",
```

Comment:
If you're bumping types, can you keep the dependencies themselves in sync? E.g. I would expect `fs-extra` to have the same semver range as its type definitions.

Reply:
Will do.